docs/io/fits/appendix/faq.rst

```diff
@@ -232,20 +228,8 @@ is likely to cause more difficulties than it's worth.
 How do I open a very large image that won't fit in memory?
 ----------------------------------------------------------
 
-In PyFITS, prior to version 3.1, when the data portion of an HDU is accessed,
-the data is read into memory in its entirety.  For example::
-
-    >>> hdul = pyfits.open('myimage.fits')
-    >>> hdul[0].data
-    ...
-
-reads the entire image array from disk into memory.  For very large images or
-tables this is clearly undesirable, if not impossible given the available
-resources.
-
-However, `astropy.io.fits.open` has an option to access the data portion of an
-HDU by memory mapping using `mmap`_.  In both Astropy and newer versions of
-PyFITS this is used by *default*.
+`astropy.io.fits.open` has an option to access the data portion of an
+HDU by memory mapping using `mmap`_.  In Astropy this is used by *default*.
```

Comment:
I would remove the italics for default, don't think it's really needed. If you want to keep it, I think the 'by' should be in italics too.